client/client_main/src/lib/GmailList/EmailListItem.jsx

```diff
@@ -1,18 +1,21 @@
-import formatDate from '../utils/dateFormatter';
-
-const options1 = {
-  is12hour: true,
-  dateCode: 'en-US',
-};
+import formatDate from '../utils/dateFormatter.js';
```

Comment:
Need a profile pic generator or something

Reply:
[#3](https://github.com/saphalpdyl/Simple-Email-App/issues/3)

This issued needs to be fixed first